src/lib/components/LanguageSelect/index.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview A select for choosing language option.
+ */
+
+import {html} from 'lit-element';
+import {BaseStateElement} from '../BaseStateElement';
+import {store} from '../../store';
+import {setLanguage, checkUserPreferredLanguage} from '../../actions';
+
+const languageNames = {
```

Comment:
Maybe both this and the array of locales used by isValidLang should be defined in a constants file that both actions.js and LanguageSelect can pull from? Then if we add another language we only need to update in one place.